src/export/digitalPlanning/model.test.ts

```diff
@@ -0,0 +1,103 @@
+import { Passport } from "../../models/passport";
+import { mockProposedLDCPassportData } from "../oneApp/mocks/passport";
+import { DigitalPlanning } from "./model"
+
+const mockPassport = new Passport({
+  data: {
+    ...mockProposedLDCPassportData,
+  },
+});
+
+describe("DigitalPlanning", () => {
+  describe("getPayload", () => {
+    it("should return valid payload", () => {
+      const instance = new DigitalPlanning({
+        sessionId: "session123",
+        passport: mockPassport,
+        templateNames: ["template1", "template2"],
+      });
+
+      const payload = instance.getPayload();
+
+      expect(payload).toEqual(instance.payload);
+    });
+
+    describe("invalid payloads", () => {
```

Comment:
I'll admit that this test suite is verging on "let's test an external library" territory but I think it's useful as a sense check that what we're aiming for here will work as expected.